README.md

```diff
@@ -44,7 +44,7 @@ To set up this notes app, complete the following tasks:
     [default]
     region = us-west-2
     ```
-
+For more information on the location of the credential and configuration files, see the [AWS documentation on file locations](https://docs.aws.amazon.com/sdkref/latest/guide/file-location.html).
```

Comment:
The main link to setting credentials file is already provided above.
```suggestion
```

https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html